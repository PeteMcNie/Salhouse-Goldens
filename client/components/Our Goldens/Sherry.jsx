import React from 'react'

import Image from 'react-bootstrap/Image'
import Table from 'react-bootstrap/Table'

import NavbarComp from '../NavbarComp'

function Sherry () {
  return (
    <>
      <NavbarComp />
      <div className='belowNavbar'>
        <h2>Sherry</h2>
      </div>
      <div>
        <h3>CH Golden Light at Salhouse CDX (Sherry)</h3>
        <p>Sire: CH Sandlewood Gamble-with-Gold</p>
        <p>Dam: CH Salhouse Sunchaser</p>
        <p>Whelped: 7 April 2010</p>
        <p>Hips: 8:8 Elbows: 1(a):B  Eyes: Clear 27/5/2014</p>
        <div>
          <Image src="images/sherry1.jpg" fluid />
        </div>
      </div>
      <div>
        <p>
          Sherry is the newest addition to our furry family. She is a very sweet puppy who was chosen for her friendly nature,
          waggy tail and, hopefully, suitability to be a show dog. She has good bone structure at this stage. She has upset the
          rest of the dogs in our family temporarily, but that will settle down once she has grown up a bit and especially when
          she loses her needle-like baby teeth and grows some proper teeth. She is quick to learn and at 10 weeks of age can sit,
          down, wait, stand, come to command. However, she can&apos;t let go of her mother&apos;s leg when called so there&apos;s still some
          training to do there.
        </p>
        <p>
          Sherry is now 17 weeks old and enjoying life to the full. She has learnt to swim and loves running around on the beach.
          She is a real retriever as she always brings us something in her mouth when she comes to see us - either a slipper, sock,
          toy, anything. Yesterday I dropped a birthday card on the floor and she picked it up and gave it to me! Sherry will be going to her first show on 8 August so it will be interesting to see how she gets on there.
        </p>
        <p>
          Sherry has achieved some successes as Baby Puppy in Breed and in the middle of August at 4 and a half months old, she went
          to the Auckland Golden Retriever shows and won Baby Puppy in Show at both shows. Critique for one of the shows was: Good
          size young lady sweetest of heads, good pigment, kind eye, well angulated at front and rear, dead level topline, has good
          overall balance, moved out well for one so young, shows good promise.
        </p>
        <p>
          Sherry has now moved on to Puppy class and has had one or two successes at the end of 2010.  In Hastings in early December
          she won two Puppy of Breeds.  She is developing really nicely. (See 8 month photo below.)
        </p>
        <p>
          Sherry has also developed her own wonderful personality and one of her most endearing traits is that she always  has
          something in her mouth when she comes to see you.  If she is rushing up to greet you and doesn&apos;t have anything in her
          mouth she stops dead, and you can see the thought processes - nothing in mouth, must get something to bring as a gift!
          So sweet.  She has also taken up putting her nose in everything, although she denies it of course (see photos below).
        </p>
        <p>
        Sherry has now got a wonderful trick - when I come in from outside the house she immediately runs off to find my slippers
        and brings them to me one at a time. You can see her thinking when I come in that she has got to find the slippers, without
        me even telling her. Sometimes when I come in from downstairs she is already waiting at the top of the stairs with one of
        them - she just can&apos;t wait to give them to me.
        </p>
        <p>
          September 2013 - Sherry has been competing in obedience competitions for about a year and a half and has just won out of
          Special Beginners. I&apos;m so proud of her. Photo and news on the news page.
        </p>
        <p>
          March 2014 - Two weeks after her mum, Shadow, became a Champion at Wanganui, we set off for four more shows at Hastings.
          The weather was lovely, the company was great and the mood seemed good. On Saturday morning we went into the show ring
          and Sherry got into the line up. She was running really nicely and the judge looked and looked and all of a sudden pointed
          to Sherry for the challenge! It took a moment to dawn on me... we have finally made another Champion... yeah. What excitement
          that was. We were on Cloud 9. We had a lovely boozy shout at lunch time to celebrate our success with friends who have
          supported us throughout the years we have been showing. It was a really nice time and something I won&apos;t forget. To top
          the weekend off, on the Sunday Shadow took the challenge in the morning show and Sherry took another challenge in the
          afternoon show - so what a great effort that was from both the girls. We didn&apos;t get home till after 9pm that day!
        </p>
        <p>
          May 2014 - Well, what a year this is becoming for Sherry. She entered her first CD for obedience in Wanganui on 3 May
          2014 and passed with flying colours gaining her CDX obedience title. We love Wanganui!
        </p>
        <p>
          February 2015 - Sherry and I finally got it together, and after nearly a year in Novice obedience we won one! We only
          lost .5 out of 100 and heat 39 other dogs! Go my sweet girl!
        </p>
        <p>
            March 2015 - We did it again a couple of weeks later - won another Novice competition and so now Sherry is official a
            Test A dog having won the two Novice tests to qualify for Test A.
        </p>
      </div>
      <div>
        <Table bordered size='sm' responsive="sm">
          <thead>
            <tr>
              <th colSpan='3'>Pedigree of CH Golden Light at Salhouse CDX (Sherry)</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>Parents</td>
              <td>Grandparents</td>
              <td>Great Grandparents</td>
            </tr>
            <tr>
              <td rowSpan='4'>Sire: CH Sandlewood Gamble-with-Gold</td>
              <td rowSpan='2'>Goldfleece Go for Gold at Sandlewood (Imp Aust)</td>
              <td>Aust CH Stanroph Shared Assets (Imp UK)</td>
            </tr>
            <tr>
              <td>Dobro Free as a Bird</td>
            </tr>
            <tr>
              <td rowSpan='2'>CH Sandlewood Charlies Angel</td>
              <td>CH McIlroy Millennium Choir Boy at Sandlewood</td>
            </tr>
            <tr>
              <td>CH Sandlewood Cinderella</td>
            </tr>
            <tr>
              <td rowSpan='4'>Dam: CH Salhouse Sunchaser</td>
              <td rowSpan='2'>NZ CH Santamaria The Patriot (Imp Aust)</td>
              <td>Aust CH Strathraer Nordic Star</td>
            </tr>
            <tr>
              <td>Aust CH Santamaria Anna Leise</td>
            </tr>
            <tr>
              <td rowSpan='2'>Sanastasia Sth&apos;n Star Chara</td>
              <td>CH Seacove Southern Cross</td>
            </tr>
            <tr>
              <td>Sanastasia Treasured Teah CDX</td>
            </tr>
          </tbody>
        </Table>
      </div>
      <hr />
      <div>
        <p>Photos:</p>
        <Image src="images/sherry2.jpg" fluid />
        <p>Sherry winning her second Novice test at Kapiti March 2015</p>
        <Image src="images/sherry3.jpg" fluid />
        <p>Sherry winning her first Novice test at Levin Feb 2015</p>
        <Image src="images/sherry4.jpg" fluid />
        <Image src="images/sherry5.jpg" fluid />
        <p>Cool dude Sherry (left). Put him down Sherry (right).</p>
        <Image src="images/sherry6.jpg" fluid />
        <Image src="images/sherry7.jpg" fluid />
        <p>I didn&apos;t put my nose in the paint pot (left) or dig in the garden (right), honest!</p>
        <Image src="images/sherry8.jpg" fluid />
        <Image src="images/sherry9.jpg" fluid />
        <Image src="images/sherry10.jpg" fluid />
        <p>Sherry with various items in her mouth - apple (left); slipper (centre); toy (right).</p>
        <Image src="images/sherry11.jpg" fluid />
        <p>Sherry at 10 months</p>
        <Image src="images/sherry12.jpg" fluid />
        <p>Sherry at 8 months</p>
        <Image src="images/sherry13.jpg" fluid />
        <p>Sherry at 6 months</p>
        <Image src="images/sherry14.jpg" fluid />
        <p>Sherry with her winnings from the Golden Retriever Club Show held in Auckland in August 2010.</p>
        <Image src="images/sherry15.jpg" fluid />
        <p>Sherry sharing her mum&apos;s bed!</p>
        <Image src="images/sherry16.jpg" fluid />
        <p>Sherry enjoys chasing Kula at the beach (15 weeks).</p>
        <Image src="images/sherry17.jpg" fluid />
        <p>Jeff and Sherry at the top of Sugarloaf Rock (14 weeks)!</p>
        <Image src="images/sherry18.jpg" fluid />
        <p>Sherry at the beach at 12 weeks.</p>
        <Image src="images/sherry19.jpg" fluid />
        <p>Sherry with a toy at 9 weeks old.</p>
        <Image src="images/sherry20.jpg" fluid />
        <p>Sherry and Shadow cuddle up.</p>
        <Image src="images/sherry21.jpg" fluid />
        <p>Sherry enjoys a kip in the sun.</p>
        <Image src="images/sherry22.jpg" fluid />
        <p>Sherry at 12 days old, playing with a ball already!</p>
        <Image src="images/sherry23.jpg" fluid />
        <p>Sherry at 1 day old</p>
      </div>
    </>
  )
}

export default Sherry
