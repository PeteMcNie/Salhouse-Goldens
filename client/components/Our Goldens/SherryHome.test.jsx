import React from 'react'
import { render } from '@testing-library/react'

import Sherry from './SherryHome'

xtest('Sherry component matches snapshot', () => {
  const { container } = render(<Sherry />)
  expect(container).toMatchSnapshot()
})
