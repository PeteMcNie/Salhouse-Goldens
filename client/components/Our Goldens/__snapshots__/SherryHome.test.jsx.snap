// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sherry component matches snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-light bg-light fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo2.png"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <nav
    class="belowNavbar nav nav-tabs"
    role="tablist"
  >
    <a
      aria-controls="controlled-tab-example-tabpane-shadowHome"
      aria-selected="true"
      class="nav-item nav-link active"
      data-rb-event-key="shadowHome"
      href="#"
      id="controlled-tab-example-tab-shadowHome"
      role="tab"
    >
      Sherry
    </a>
    <a
      aria-controls="controlled-tab-example-tabpane-ShadowLitter"
      aria-disabled="true"
      aria-selected="false"
      class="nav-item nav-link disabled"
      data-rb-event-key="ShadowLitter"
      href="#"
      id="controlled-tab-example-tab-ShadowLitter"
      role="tab"
      tabindex="-1"
    >
      Sherry's Litter
    </a>
    <a
      aria-controls="controlled-tab-example-tabpane-breedShowAchievements"
      aria-selected="false"
      class="nav-item nav-link"
      data-rb-event-key="breedShowAchievements"
      href="#"
      id="controlled-tab-example-tab-breedShowAchievements"
      role="tab"
      tabindex="-1"
    >
      Breed Show Achievements
    </a>
    <a
      aria-controls="controlled-tab-example-tabpane-obedienceShowAchievements"
      aria-selected="false"
      class="nav-item nav-link"
      data-rb-event-key="obedienceShowAchievements"
      href="#"
      id="controlled-tab-example-tab-obedienceShowAchievements"
      role="tab"
      tabindex="-1"
    >
      Obedience Show Achievements
    </a>
  </nav>
  <div
    class="tab-content"
  >
    <div
      aria-hidden="false"
      aria-labelledby="controlled-tab-example-tab-shadowHome"
      class="fade tab-pane active show"
      id="controlled-tab-example-tabpane-shadowHome"
      role="tabpanel"
    >
      <div>
        <h2>
          Sherry
        </h2>
      </div>
      <div>
        <h3>
          CH Golden Light at Salhouse CDX (Sherry)
        </h3>
        <p>
          Sire: CH Sandlewood Gamble-with-Gold
        </p>
        <p>
          Dam: CH Salhouse Sunchaser
        </p>
        <p>
          Whelped: 7 April 2010
        </p>
        <p>
          Hips: 8:8 Elbows: 1(a):B  Eyes: Clear 27/5/2014
        </p>
        <div>
          <img
            class="img-fluid"
            src="images/sherry1.jpg"
          />
        </div>
      </div>
      <div>
        <p>
          Sherry is the newest addition to our furry family. She is a very sweet puppy who was chosen for her friendly nature, waggy tail and, hopefully, suitability to be a show dog. She has good bone structure at this stage. She has upset the rest of the dogs in our family temporarily, but that will settle down once she has grown up a bit and especially when she loses her needle-like baby teeth and grows some proper teeth. She is quick to learn and at 10 weeks of age can sit, down, wait, stand, come to command. However, she can't let go of her mother's leg when called so there's still some training to do there.
        </p>
        <p>
          Sherry is now 17 weeks old and enjoying life to the full. She has learnt to swim and loves running around on the beach. She is a real retriever as she always brings us something in her mouth when she comes to see us - either a slipper, sock, toy, anything. Yesterday I dropped a birthday card on the floor and she picked it up and gave it to me! Sherry will be going to her first show on 8 August so it will be interesting to see how she gets on there.
        </p>
        <p>
          Sherry has achieved some successes as Baby Puppy in Breed and in the middle of August at 4 and a half months old, she went to the Auckland Golden Retriever shows and won Baby Puppy in Show at both shows. Critique for one of the shows was: Good size young lady sweetest of heads, good pigment, kind eye, well angulated at front and rear, dead level topline, has good overall balance, moved out well for one so young, shows good promise.
        </p>
        <p>
          Sherry has now moved on to Puppy class and has had one or two successes at the end of 2010.  In Hastings in early December she won two Puppy of Breeds.  She is developing really nicely. (See 8 month photo below.)
        </p>
        <p>
          Sherry has also developed her own wonderful personality and one of her most endearing traits is that she always  has something in her mouth when she comes to see you.  If she is rushing up to greet you and doesn't have anything in her mouth she stops dead, and you can see the thought processes - nothing in mouth, must get something to bring as a gift! So sweet.  She has also taken up putting her nose in everything, although she denies it of course (see photos below).
        </p>
        <p>
          Sherry has now got a wonderful trick - when I come in from outside the house she immediately runs off to find my slippers and brings them to me one at a time. You can see her thinking when I come in that she has got to find the slippers, without me even telling her. Sometimes when I come in from downstairs she is already waiting at the top of the stairs with one of them - she just can't wait to give them to me.
        </p>
        <p>
          September 2013 - Sherry has been competing in obedience competitions for about a year and a half and has just won out of Special Beginners. I'm so proud of her. Photo and news on the news page.
        </p>
        <p>
          March 2014 - Two weeks after her mum, Shadow, became a Champion at Wanganui, we set off for four more shows at Hastings. The weather was lovely, the company was great and the mood seemed good. On Saturday morning we went into the show ring and Sherry got into the line up. She was running really nicely and the judge looked and looked and all of a sudden pointed to Sherry for the challenge! It took a moment to dawn on me... we have finally made another Champion... yeah. What excitement that was. We were on Cloud 9. We had a lovely boozy shout at lunch time to celebrate our success with friends who have supported us throughout the years we have been showing. It was a really nice time and something I won't forget. To top the weekend off, on the Sunday Shadow took the challenge in the morning show and Sherry took another challenge in the afternoon show - so what a great effort that was from both the girls. We didn't get home till after 9pm that day!
        </p>
        <p>
          May 2014 - Well, what a year this is becoming for Sherry. She entered her first CD for obedience in Wanganui on 3 May 2014 and passed with flying colours gaining her CDX obedience title. We love Wanganui!
        </p>
        <p>
          February 2015 - Sherry and I finally got it together, and after nearly a year in Novice obedience we won one! We only lost .5 out of 100 and heat 39 other dogs! Go my sweet girl!
        </p>
        <p>
          March 2015 - We did it again a couple of weeks later - won another Novice competition and so now Sherry is official a Test A dog having won the two Novice tests to qualify for Test A.
        </p>
      </div>
      <div>
        <div
          class="table-responsive-sm"
        >
          <table
            class="table table-sm table-bordered"
          >
            <thead>
              <tr>
                <th
                  colspan="3"
                >
                  Pedigree of CH Golden Light at Salhouse CDX (Sherry)
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  Parents
                </td>
                <td>
                  Grandparents
                </td>
                <td>
                  Great Grandparents
                </td>
              </tr>
              <tr>
                <td
                  rowspan="4"
                >
                  Sire: CH Sandlewood Gamble-with-Gold
                </td>
                <td
                  rowspan="2"
                >
                  Goldfleece Go for Gold at Sandlewood (Imp Aust)
                </td>
                <td>
                  Aust CH Stanroph Shared Assets (Imp UK)
                </td>
              </tr>
              <tr>
                <td>
                  Dobro Free as a Bird
                </td>
              </tr>
              <tr>
                <td
                  rowspan="2"
                >
                  CH Sandlewood Charlies Angel
                </td>
                <td>
                  CH McIlroy Millennium Choir Boy at Sandlewood
                </td>
              </tr>
              <tr>
                <td>
                  CH Sandlewood Cinderella
                </td>
              </tr>
              <tr>
                <td
                  rowspan="4"
                >
                  Dam: CH Salhouse Sunchaser
                </td>
                <td
                  rowspan="2"
                >
                  NZ CH Santamaria The Patriot (Imp Aust)
                </td>
                <td>
                  Aust CH Strathraer Nordic Star
                </td>
              </tr>
              <tr>
                <td>
                  Aust CH Santamaria Anna Leise
                </td>
              </tr>
              <tr>
                <td
                  rowspan="2"
                >
                  Sanastasia Sth'n Star Chara
                </td>
                <td>
                  CH Seacove Southern Cross
                </td>
              </tr>
              <tr>
                <td>
                  Sanastasia Treasured Teah CDX
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
      <hr />
      <div>
        <p>
          Photos:
        </p>
        <img
          class="img-fluid"
          src="images/sherry2.jpg"
        />
        <p>
          Sherry winning her second Novice test at Kapiti March 2015
        </p>
        <img
          class="img-fluid"
          src="images/sherry3.jpg"
        />
        <p>
          Sherry winning her first Novice test at Levin Feb 2015
        </p>
        <img
          class="img-fluid"
          src="images/sherry4.jpg"
        />
        <img
          class="img-fluid"
          src="images/sherry5.jpg"
        />
        <p>
          Cool dude Sherry (left). Put him down Sherry (right).
        </p>
        <img
          class="img-fluid"
          src="images/sherry6.jpg"
        />
        <img
          class="img-fluid"
          src="images/sherry7.jpg"
        />
        <p>
          I didn't put my nose in the paint pot (left) or dig in the garden (right), honest!
        </p>
        <img
          class="img-fluid"
          src="images/sherry8.jpg"
        />
        <img
          class="img-fluid"
          src="images/sherry9.jpg"
        />
        <img
          class="img-fluid"
          src="images/sherry10.jpg"
        />
        <p>
          Sherry with various items in her mouth - apple (left); slipper (centre); toy (right).
        </p>
        <img
          class="img-fluid"
          src="images/sherry11.jpg"
        />
        <p>
          Sherry at 10 months
        </p>
        <img
          class="img-fluid"
          src="images/sherry12.jpg"
        />
        <p>
          Sherry at 8 months
        </p>
        <img
          class="img-fluid"
          src="images/sherry13.jpg"
        />
        <p>
          Sherry at 6 months
        </p>
        <img
          class="img-fluid"
          src="images/sherry14.jpg"
        />
        <p>
          Sherry with her winnings from the Golden Retriever Club Show held in Auckland in August 2010.
        </p>
        <img
          class="img-fluid"
          src="images/sherry15.jpg"
        />
        <p>
          Sherry sharing her mum's bed!
        </p>
        <img
          class="img-fluid"
          src="images/sherry16.jpg"
        />
        <p>
          Sherry enjoys chasing Kula at the beach (15 weeks).
        </p>
        <img
          class="img-fluid"
          src="images/sherry17.jpg"
        />
        <p>
          Jeff and Sherry at the top of Sugarloaf Rock (14 weeks)!
        </p>
        <img
          class="img-fluid"
          src="images/sherry18.jpg"
        />
        <p>
          Sherry at the beach at 12 weeks.
        </p>
        <img
          class="img-fluid"
          src="images/sherry19.jpg"
        />
        <p>
          Sherry with a toy at 9 weeks old.
        </p>
        <img
          class="img-fluid"
          src="images/sherry20.jpg"
        />
        <p>
          Sherry and Shadow cuddle up.
        </p>
        <img
          class="img-fluid"
          src="images/sherry21.jpg"
        />
        <p>
          Sherry enjoys a kip in the sun.
        </p>
        <img
          class="img-fluid"
          src="images/sherry22.jpg"
        />
        <p>
          Sherry at 12 days old, playing with a ball already!
        </p>
        <img
          class="img-fluid"
          src="images/sherry23.jpg"
        />
        <p>
          Sherry at 1 day old
        </p>
      </div>
    </div>
    <div
      aria-hidden="true"
      aria-labelledby="controlled-tab-example-tab-ShadowLitter"
      class="fade tab-pane"
      id="controlled-tab-example-tabpane-ShadowLitter"
      role="tabpanel"
    >
      <div>
        <h2>
          Sherry's Litter 2006
        </h2>
        <h1>
          Copy Template from Kula'sLitter
        </h1>
      </div>
    </div>
    <div
      aria-hidden="true"
      aria-labelledby="controlled-tab-example-tab-breedShowAchievements"
      class="fade tab-pane"
      id="controlled-tab-example-tabpane-breedShowAchievements"
      role="tabpanel"
    >
      <div>
        <h2>
          Show achievements
        </h2>
        <p>
          This page will document Sherry's show achievements - CH Golden Light at Salhouse.
        </p>
      </div>
      <div>
        <div
          class="table-responsive-sm"
        >
          <table
            class="table table-sm table-bordered"
          >
            <thead>
              <tr>
                <th
                  colspan="3"
                >
                  Challenge Certificates Awarded
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  Date
                </td>
                <td>
                  Show
                </td>
                <td>
                  Judge
                </td>
              </tr>
              <tr>
                <td>
                  29 January 2011
                </td>
                <td>
                  Nelson District Kennel Association
                </td>
                <td>
                  C Butler (NSW)
                </td>
              </tr>
              <tr>
                <td>
                  30 January 2011
                </td>
                <td>
                  Nelson District Kennel Association
                </td>
                <td>
                  L Butler (NSW)
                </td>
              </tr>
              <tr>
                <td>
                  5 November 2011
                </td>
                <td>
                  Southland Kennel Association
                </td>
                <td>
                  Justine Blyth (Tasmania)
                </td>
              </tr>
              <tr>
                <td>
                  6 November 2011
                </td>
                <td>
                  Otago-Southland Gundog Society
                </td>
                <td>
                  J Ellem (NSW)
                </td>
              </tr>
              <tr>
                <td>
                  1 April 2012
                </td>
                <td>
                  Napier Kennel Centre
                </td>
                <td />
              </tr>
              <tr>
                <td>
                  16 February 2013
                </td>
                <td>
                  Horowhenua Kennel Association
                </td>
                <td>
                  Honey Glendinning (Canada)
                </td>
              </tr>
              <tr>
                <td>
                  13 July 2013
                </td>
                <td>
                  Otago Kennel Association
                </td>
                <td>
                  Lynette Brown (NSW)
                </td>
              </tr>
              <tr>
                <td>
                  29 March 2014
                </td>
                <td>
                  Wairoa Kennel Association
                </td>
                <td>
                  Max Martin (ACT)
                </td>
              </tr>
              <tr>
                <td>
                  30 March 2014
                </td>
                <td>
                  Napier Kennel Centre
                </td>
                <td>
                  Lena Stalhandske (NSW)
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
      <hr />
      <div>
        <img
          class="img-fluid"
          src="images/sherryShowAchievements.jpg"
        />
        <p>
          Sherry wins Puppy of Group at Horowhenua Kennel Society show 13 Feburary 2011.
        </p>
        <img
          class="img-fluid"
          src="images/sherry14.jpg"
        />
        <p>
          Sherry with her Baby Puppy in Show awards from Auckland in August 2010.
        </p>
      </div>
    </div>
    <div
      aria-hidden="true"
      aria-labelledby="controlled-tab-example-tab-obedienceShowAchievements"
      class="fade tab-pane"
      id="controlled-tab-example-tabpane-obedienceShowAchievements"
      role="tabpanel"
    >
      <div>
        <h2>
          Obedience show achievements
        </h2>
      </div>
      <div>
        <div>
          <div
            class="table-responsive-sm"
          >
            <table
              class="table table-sm table-borderless"
            >
              <thead>
                <tr>
                  <th
                    colspan="2"
                  >
                    Obedience show achievements
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    Date
                  </td>
                  <td>
                    Show Details
                  </td>
                </tr>
                <tr>
                  <td>
                    25 February 2012
                  </td>
                  <td>
                    <p>
                      Sherry competed in her first Championship obedience test in Special Beginners and achieved 2nd place on 72 points. Very proud of her.
                    </p>
                    <img
                      class="img-fluid"
                      src="images/so1.jpg"
                    />
                    <img
                      class="img-fluid"
                      src="images/so2.jpg"
                    />
                  </td>
                </tr>
                <tr>
                  <td>
                    26 February 2012
                  </td>
                  <td>
                    <p>
                      Sherry got 2nd place with 69 points.
                    </p>
                    <img
                      class="img-fluid"
                      src="images/so3.jpg"
                    />
                  </td>
                </tr>
                <tr>
                  <td>
                    10 March 2012
                  </td>
                  <td>
                    4th place - 70.5 points, Waikanae
                  </td>
                </tr>
                <tr>
                  <td>
                    11 March 2012
                  </td>
                  <td>
                    <p>
                      2nd place - 71 points, Waikanae
                    </p>
                    <div>
                      <div
                        class="embed-responsive embed-responsive-16by9"
                      >
                        <embed
                          class="embed-responsive-item"
                          src="https://www.youtube.com/embed/QL6PinJ3p64"
                        />
                      </div>
                    </div>
                  </td>
                </tr>
                <tr>
                  <td>
                    8 September 2013
                  </td>
                  <td>
                    <p>
                      1st place - 72 points, Hawke's Bay - (won out of Special Beginners)
                    </p>
                    <img
                      class="img-fluid"
                      src="images/so4.jpg"
                    />
                  </td>
                </tr>
                <tr>
                  <td>
                    3 May 2014
                  </td>
                  <td>
                    Sherry gained her CDX title gaining 161.5 points out of 170! Wanganui
                  </td>
                </tr>
                <tr>
                  <td>
                    21 February 2015
                  </td>
                  <td>
                    <p>
                      Photo below shows Sherry winning her first Novice after nearly a year in that class. She scored 99.5 out of 100, Horowhenua Obedience, Levin
                    </p>
                    <p>
                      One more win in Novice and then she has to go up to Test A.
                    </p>
                    <img
                      class="img-fluid"
                      src="images/so5.jpg"
                    />
                  </td>
                </tr>
                <tr>
                  <td>
                    21 February 2015
                  </td>
                  <td>
                    <p>
                      Photo below shows Sherry winning her second Novice competition scoring 97 points, Kapiti Obedience, Waikanae 
                    </p>
                    <p>
                      This qualifies her to compete in Test A competitions now.
                    </p>
                    <img
                      class="img-fluid"
                      src="images/so6.jpg"
                    />
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
